src/utilities/theme-utils.ts

```diff
@@ -0,0 +1,28 @@
+// example from:
+// https://stackoverflow.com/questions/5560248/programmatically-lighten-or-darken-a-hex-color-or-rgb-and-blend-colors
```

Comment:
```suggestion
// https://stackoverflow.com/a/62640342
```
Share link to actual answer that appears to be the source of the code in question. It's a long thread...

Reply:
Fixed.